infra/command/gen-coverage-report

```diff
@@ -67,7 +67,7 @@ done
   "${CANDIDATES[@]}"
 
 
-opencl_files=($(find ./runtime/onert/backend/gpu_cl/open_cl/ \( -name "*.cc" -o -name "*.h" \) -exec realpath {} \; ))
+opencl_files=($(find ./runtime/onert/backend/gpu_cl/open_cl/ \( -name "*.cc" -o -name "*.h" \) -exec realpath {} \; | true))
```

Comment:
We need this fix to get trix backend only coverage.

Reply:
@hseok-oh I do not understand the comment yet.